lib/upsert/merge_function.rb

```diff
@@ -31,12 +31,13 @@ def unique_name(table_name, selector_keys, setter_keys)
     attr_reader :selector_keys
     attr_reader :setter_keys
 
-    def initialize(controller, selector_keys, setter_keys, assume_function_exists)
+    def initialize(controller, selector_keys, setter_keys, assume_function_exists, use_native_upsert = nil)
```

Comment:
right - externally it's an option (i suggested `disable_native`), but internally it's not an option